support/mock/mock_syscaller.go

```diff
@@ -0,0 +1,54 @@
+package mock
+
+import (
+	addr "github.com/filecoin-project/go-address"
+	"github.com/ipfs/go-cid"
+
+	"github.com/filecoin-project/specs-actors/actors/abi"
+	"github.com/filecoin-project/specs-actors/actors/crypto"
+	"github.com/filecoin-project/specs-actors/actors/runtime"
+)
+
+type VerifyFunc func(signature crypto.Signature, signer addr.Address, plaintext []byte) bool
+type HasherFunc func(data []byte) []byte
+
+type syscaller struct {
+	Verifier VerifyFunc
+	Hasher   HasherFunc
+}
+
+// Interface methods
+func (s *syscaller) VerifySignature(sig crypto.Signature, signer addr.Address, plaintext []byte) bool {
+	if s.Verifier == nil {
+		panic("set me")
```

Comment:
I like the general way you've set this up, but please replace these panics with `s.t.Fatalf("no verifier set")` etc.

Reply:
This has a side effect the linter pointed out, which is, the builder does a copy of the runtime, and since syscalls would take testing.T in them, now those objects have RWMutexes, so it flags it as a copylock.  I’ll leave it as a panic but helpfully emit the function name in the panic message.